tests/python/gaia-ui-tests/gaiatest/apps/ring_tone/app.py

```diff
@@ -9,13 +9,17 @@
 
 class RingTone(Base):
     name = 'Ringtones'
+    origin = 'app://ringtones.gaiamobile.org'
```

Comment:
ah, then I should wait until 1188926 is merged, because without it i can't run it in arabic locale now